docs/source/User-guide/Post-process.rst

```diff
@@ -1,4 +1,92 @@
 Post-process
 =============
 
-TBC
+MapReader post-processing's sub-package currently contains one method for post-processing the predictions from your model based on the idea that features such as railways, roads, coastlines, etc. are continuous and so patches with these labels should be found near to other patches also with these labels.
+
+For example, if a patch is predicted to be a railspace, but is surrounded by patches predicted to be non-railspace, then it is likely that the railspace patch is a false positive.
```

Comment:
I guess you could be even more explicit and say: "The current method checks whether any of the 8 surrounding patches have the same label as a given patch (e.g. railspace), and if not, assumes this to be a false positive".

Perhaps could also mention: "Future releases may add functionality to create custom filter rules for your use case"